packages/messages/src/browser/notifications-message-client.ts

```diff
@@ -73,7 +73,8 @@ export class NotificationsMessageClient extends MessageClient {
             icon,
             text,
             actions,
-            timeout
+            timeout,
+            timeoutFn
```

Comment:
What do you think of passing `() => onCloseFn(undefined)` as value for `timeoutFn` here?

Reply:
Which would actually solve another issue, as far as see the returned promise never resolves (as undefined) on timeout events, right?